common/views/components/MetaUnit/MetaUnit.js

```diff
@@ -80,7 +74,10 @@ const List = ({ list }) => {
   return (
     list.length > 0 && (
       <ul
-        className={`${spacing({ s: 2 }, { margin: ['bottom'] })} ${spacing(
+        className={`plain-list ${spacing(
+          { s: 2 },
+          { margin: ['bottom'] }
+        )} ${spacing(
```

Comment:
@wellcometrust/experience-devs What do we think about using the `classNames` function universally?

Looking at this also makes me wonder if we could do some sort of analysis on the different spacings we use, and start working with a more standardised grid...